test/awesompleteShared.js

```diff
@@ -41,5 +41,12 @@ var shared = {
 			expect(li.getAttribute('aria-selected')).toBe('true');
       expect(shared.awesompleter.status.textContent).toBe(li.textContent);
 		}
+  },
+  getListFromResults: function (awesompleter){
```

Comment:
Looks like an awesomplete API needs some tweaks (not only for tests). See my comment here https://github.com/LeaVerou/awesomplete/issues/16725#issuecomment-100992477

Thoughts?